poly-commitment/src/evaluation_proof.rs

```diff
@@ -3,17 +3,33 @@
 //! Zero-Knowledge Arguments for Arithmetic Circuits in the Discrete Log
 //! Setting](https://eprint.iacr.org/2016/263)
 
-use crate::{commitment::*, srs::SRS, PolynomialsToCombine};
-use ark_ec::AffineRepr;
-use ark_ff::{FftField, Field, One, PrimeField, Zero};
-use ark_poly::{univariate::DensePolynomial, DenseUVPolynomial, EvaluationDomain, Evaluations};
+use crate::{
```

Comment:
No changes here. Mostly moving from `srs`. I did remove one method (`max_degree`) and remove some duplicated methods.